engine/src/test/java/io/zeebe/engine/processor/workflow/handlers/element/ElementActivatedHandlerTest.java

```diff
@@ -1,70 +0,0 @@
-/*
```

Comment:
I decided to remove the unit tests of the handlers instead of making them work again. I think these unit tests don't give us much value because they mock too many dependencies and rely heavily on the behavior of the dependencies. 

In case of the handlers, I prefer integration tests with the engine. As far as I know, we cover each of these cases in an integration test.